manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -35,6 +35,8 @@ gateways:
 
     cpu:
       targetAverageUtilization: 80
+    memory: {}
```

Comment:
Can we only add this to the `manifests/charts/gateway`? It's not recommended to use istio-ingress and istio-egress anymore for a long time. See https://github.com/istio/istio/tree/master/manifests/charts/gateway#legacy-gateway-helm-charts

Reply:
I didn't know that. It's better to erase it. Thank you :)